app/models/linked_case.rb

```diff
@@ -0,0 +1,74 @@
+class LinkedCase < ApplicationRecord
+  self.inheritance_column = :_unused
```

Comment:
would have preferred to see the type column called `link_type`, then there would be no need to disable the STI

Reply:
I wanted do have the option to enable STI without having to rename the column, etc, in the future, and I also thought it read well as 'link.type'.